iaac/terraform/kubeflow-components/central-dashboard/main.tf

```diff
@@ -0,0 +1,5 @@
+module "helm_addon" {
+  source            = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons/helm-addon?ref=v4.9.0"
```

Comment:
is // intentional? 
`terraform-aws-eks-blueprints//modules`

Reply:
yes, specifies path in module to load